src/pages/components/tabs/usage.mdx

```diff
@@ -280,6 +292,18 @@ to drive visual rhythm by spacing content in multiples of two columns and
 aligning the beginning and ending of the tab elements with content below the
 tabs when possible.
 
+Use grid aligned contained tabs when:
+
+- When there are 4 tabs or less
+- When the tablist can fit on the 2x grid easily without crowding labels
+- Labels are short and concise
+- When there are other elements on the page that can align to the tablist
+
+Depending on breakpoints, vertical tabs align to grid columns spanning 4 or 2
```

Comment:
Maybe

 "...vertical tabs align to grid columns spanning 4 or 2 columns of the 16-column grid and connect to a tab panel that can be 8 or 12 columns wide."